packages/user-interface/src/pages/ThemeDetailsPage.tsx

```diff
@@ -54,6 +54,7 @@ const ThemeDetailsPage = ({
         error={tasksError}
         titleTranslationId={null}
         tasks={tasks}
+        openInContext={true}
```

Comment:
Hier niet toch? want dit is de context :P

Reply:
Foutje, heb het aangepast!